.github/workflows/build_wheels.yml

```diff
@@ -20,9 +23,10 @@ jobs:
   build_wheels:
     name: Build wheel for ${{ matrix.os }}, Python ${{ matrix.pyver }}
     runs-on: ${{ matrix.os }}
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'build_wheels') || github.ref == 'refs/heads/master' || github.ref == 'refs/heads/next' }}
```

Comment:
This won't trigger on a tag build though.

Perhaps try this:
`github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'build_wheels')`